packages/docs/concepts/menus.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020-2022 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/general.yaml.njk
+  vars:
+    pageId: menus
+    pageTitle: Menus
+    section: Concepts
+    filePath: concepts/menus.yaml
+    content:
+      - id: alert2
+        type: Alert
+        properties:
+          type: info
+          showIcon: false
+          message: The menu items are filtered according to the roles given to the authenticated user, if a user has been authenticated.
```

Comment:
They aren't only filtered by roles, but also by access (public or authnticated). I think the ting we should say is filtered acording to pages the user is authorized to see.